reference/fleet/get-started-managed-apm-server.md

```diff
@@ -71,7 +71,7 @@ You can install only a single {{agent}} per host, which means you cannot run {{f
 4. Save and apply the settings.
```

Comment:
Should L13 (briefly) explain what a Fleet-managed APM Server is? I guess that happens in step 1: "Use Fleet in Kibana to get APM data into the Elastic Stack." Is that good enough?